config/locales/two_factor_authentication/en.yml

```diff
@@ -19,6 +19,16 @@ en:
     backup_code_header_text: Enter your backup security code
     backup_code_prompt: You can use this security code once. After you enter it,
       you’ll need to use a new key.
+    backup_codes:
+      add_another_authentication_option: '‹ Add another authentication option'
```

Comment:
Sorta curious how screen readers handle the `‹` character, though I acknowledge it's a pattern we already use elsewhere.

Reply:
This is a good question and something that I had not thought of before. I will test and report on what happens.

Reply:
So good news--the character is not announced 

https://github.com/18F/identity-idp/assets/17969963/9190d8be-26b2-4250-a9c5-b2c59fd83d36

Reply:
> So good news--the character is not announced

That's good! Thanks for testing.

Related, I also stumbled across this resource related, which seems to confirm that it shouldn't announce anything in the context of the broad sentence, including for other screen readers.

https://www.deque.com/blog/dont-screen-readers-read-whats-screen-part-1-punctuation-typographic-symbols/